spec/src/main/asciidoc/overview.asciidoc

```diff
@@ -68,101 +68,69 @@ Instances of `ManagedExecutor` and `ThreadContext` can be constructed via builde
 
 Applications should shut down instances of `ManagedExecutor` that they build after they are no longer needed. The shutdown request serves as a signal notifying the container that resources can be safely cleaned up.
 
-=== Injection of Configured Instances
+=== Example usage
 
-The `@ManagedExecutorConfig` and `@ThreadContextConfig` annotations provide the ability to configure the injection points with desired capabilities.
-With just the config annotation in place, every injection point is assigned a new instance.
+For managed executor,
 
 [source, java]
 ----
-    @Inject 
-    @ManagedExecutorConfig(maxAsync = 5, propagated = ThreadContext.SECURITY, cleared = ThreadContext.ALL_REMAINING)
-    ManagedExecutor configuredExecutor;
-    ...
-    CompletableFuture<Long> stage = configuredExecutor
-        .newIncompleteFuture()
+    CompletableFuture<Long> stage = executor.newIncompleteFuture()
         .thenApply(function)
         .thenAccept(consumer);
     stage.completeAsync(supplier);
 ----
 
-Or similarly:
+Or similarly for thread context,
 
 [source, java]
 ----
-    @Inject 
-    @ThreadContextConfig(propagated = ThreadContext.SECURITY, unchanged = {}, cleared = ThreadContext.ALL_REMAINING)
-    ThreadContext threadContext;
-    ...
     threadContext.withContextCapture(unmanagedCompletionStage)
         .thenApply(function)
         .thenAccept(consumer);
 ----
 
-=== Sharing Configured Instances
+=== Sharing Instances
 
-Configured instances of `ManagedExecutor` and `ThreadContext` are shared based on their unique name, which is assigned by the CDI qualifier, `@NamedInstance`.
-When one injection point is annotated with both a configuration and a `@NamedInstance` qualifier, the application can inject the same instance elsewhere by annotating other injection points with just the matching qualifier.
-In the example below, the `executor1` injection point defines a configuration and a name `myExec` for a configured `ManagerExecutor` instance.
-The other injection points, `executor2` and `executor3`, share the same `ManagedExecutor` instance by using the `@NamedInstance` qualifier with the name `myExec`.
+CDI application scope is a convenient mechanism for sharing instances across an application.
```

Comment:
`CDI producers combined with injection are a convenient mechanism for sharing...`

Reply:
Updating as recommended.